packages/core/examples/tabs2Example.tsx

```diff
@@ -0,0 +1,96 @@
+                key={this.state.vertical ? "vertical" : "horizontal"}
+                onChange={this.handleTabChange}
+                renderActiveTabPanelOnly={this.state.activePanelOnly}
+                vertical={this.state.vertical}
+            >
+                <Tab2 id="react" title="React">
+                    <h3>Example panel: React</h3>
+                    <p className="pt-running-text">
+                        Lots of people use React as the V in MVC. Since React makes no assumptions about the
+                        rest of your technology stack, it's easy to try it out on a small feature in an existing
+                        project.
+                    </p>
+                </Tab2>
+                <Tab2 id="angular" title={this.getTitle("Angular")}>
+                    <h3>Example panel: Angular</h3>
+                    <p className="pt-running-text">
+                        HTML is great for declaring static documents, but it falters when we try to use it for
+                        declaring dynamic views in web-applications. AngularJS lets you extend HTML vocabulary
+                        for your application. The resulting environment is extraordinarily expressive, readable,
+                        and quick to develop.
+                    </p>
+                </Tab2>
+                <Tab2 id="ember" title={this.getTitle("Ember")}>
+                    <h3>Example panel: Ember</h3>
+                    <p className="pt-running-text">
+                        Ember.js is an open-source JavaScript application framework, based on the
+                        model-view-controller (MVC) pattern. It allows developers to create scalable single-page
+                        web applications by incorporating common idioms and best practices into the framework.
+                        What is your favorite JS framework?
+                    </p>
+                    <input className="pt-input" type="text"/>
+                </Tab2>
+                <Tab2 id="backbone" disabled title="Backbone">
+                    <h3>Backbone</h3>
+                </Tab2>
+                <input className="pt-input" type="text" placeholder="Search..." />
+            </Tabs2>
+        );
+    }
+
+    protected renderOptions() {
+        return [
+            [
+                <Switch
+                    checked={this.state.vertical}
+                    label="Use vertical tabs"
+                    key="vertical"
+                    onChange={this.toggleVertical}
+                />,
+                <Switch
+                    checked={this.state.activePanelOnly}
+                    label="Render active tab panel only"
+                    key="active"
+                    onChange={this.toggleActiveOnly}
+                />,
+            ],
+        ];
+    }
+
+    private getTitle(title: string) {
+        return title + (this.state.activeTabId === title.toLowerCase() ? " (active)" : "");
```

Comment:
no we certainly don't. i added that to test something we had a unit test for in the old API, and it just worked out of the box. i'd like to refactor the example a bunch so need to review it thoroughly (too late 😄 )